code/modules/projectiles/guns/energy.dm

```diff
@@ -885,7 +885,7 @@
 	wield_delay = 1 SECONDS
 	scatter = 0
 	scatter_unwielded = 10
-	fire_delay = 0.15 SECONDS
+	fire_delay = 0.2 SECONDS
```

Comment:
Undocumented balance changes

Reply:
It isn't.
The firemode is 0.2 already, this was something I missed in the first PR.